tests/testDbLocal.py

```diff
@@ -49,17 +48,19 @@
 import time
 import unittest
 
+# third party
+import sqlalchemy
+
 # local
-from lsst.db.db import Db, DbException
-from lsst.db.utils import readCredentialFile
+from lsst.db.db import Db
+from lsst.db.mysqlUtils import *
```

Comment:
`import *` is a terrible thing and should not be used in production scripts. I'd just do `from lsst.db import mysqlUtils` and then use `mysqlUtils.method()`